tests/e2e/tidbcluster/tidbcluster.go

```diff
@@ -1345,6 +1184,15 @@ var _ = ginkgo.Describe("[tidb-operator] TiDBCluster", func() {
 			err = oa.WaitForTidbClusterReady(tc, 30*time.Minute, 15*time.Second)
 			framework.ExpectNoError(err)
 		})
+
+		ginkgo.It("Backup and Restore with TLS enabled", func() {
+			provider := framework.TestContext.Provider
+			if provider != "aws" && provider != "kind" {
+				framework.Skipf("provider is not aws or kind, skipping")
+			}
+
+			testBR("kind", ns, fw, c, genericCli, oa, cli, true)
```

Comment:
```suggestion
			testBR(provider, ns, fw, c, genericCli, oa, cli, true)
```

if `kind` is only supported, you can skip if the provider is not `kind`